File: JAVA/xtra/Duplicate.java
class Duplicate
{
public static void main(String arg[])
{
 int arr[]={1,2,3,1,5};

 for(int i=0; i<arr.length; i++)
 {
	for(int j=i+1; j<arr.length; j++)
	{
	 if(arr[i]==arr[j])
	 {
	   System.out.println("Duplicate is"+arr[i]);
	 }
	 
}
}

//		 System.out.println("no duplicate");
}
}

File: C/duplicate.c
#include<stdio.h>
#include<conio.h>
int main()
{
	int i,j,arr[10],n;
	int flag=0;
	printf("Enter the number of elements: ");
	scanf("%d", &n);
	
	for(i=0; i<n; i++)
	{
		printf("Enter the element at arr[%d]:",i+1);
		scanf("%d", &arr[i]);
	}
	
	for(i=0; i<n; i++)
	{
		for(j=i+1; j<n; j++)
		{
			if(arr[i]==arr[j] && i!=j)
			{
				flag=1;
				printf(" duplicate element found at %d and %d.", i,j);
			}
		}
	}
	if(flag==0)
	{
		printf("No duplicte element.");
	}
}
